cmd/query/app/server.go

```diff
@@ -43,15 +43,12 @@ type Server struct {
 	querySvc     *querysvc.QueryService
 	queryOptions *QueryOptions
 
-	conn          net.Listener
-	grpcConn      net.Listener
-	httpConn      net.Listener
-	cmuxServer    cmux.CMux
```

Comment:
I would've expected the import of cmux to be removed as well.

Reply:
if you do remove it, run `go mod tidy`

Reply:
done